drivers/sensor/sensor_decoders_init.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2023 Google LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/drivers/sensor.h>
+
+static int sensor_decoders_init(void)
+{
+	STRUCT_SECTION_FOREACH(sensor_decoder_api, api)
+	{
```

Comment:
bring up the bracket?

Reply:
clang-format did this :) I would prefer not to argue and just let it do it's thing so I don't have to turn off formatting for this 1 line.

Reply:
Fixed by adding the macro to `ForEachMacros` in `clang-format`